Tests/Unit/NetworkServer/LoRaDeviceTest.cs

```diff
@@ -106,33 +139,28 @@ public async Task After_Saving_Frame_Counter_Changes_Should_Not_Have_Pending_Cha
         [Fact]
         public async Task When_Initialized_New_OTAA_Device_Should_Have_All_Properties()
         {
-            var appKey = TestKeys.CreateAppKey(0xABC0200000000000, 0x09);
-            var joinEui = new JoinEui(0xABC0200000000009);
-
-            var twin = TestUtils.CreateTwin(
-                desired: new Dictionary<string, object>
+            const string gateway = "mygateway";
```

Comment:
Do we need a different gateway identifier here? Why not reuse the one from `OtaaDesiredTwinProperties.GatewayId`?

Reply:
Then the assertions on the which is the owning gateway would break. I kept the assertions from the old tests, I will give it another pass and see if I can simplify further.

Reply:
I replaced the existing usages with a separate test that explicitly tests this behavior in fe45f82239fef39ab3b7fa818a73da9e8e66b3b8